src/utils/rabbitmq.js

```diff
@@ -82,7 +82,6 @@ class Channel {
 	async consumeQueue(consumer, options) {
 		try {
 			const ch = await this.getChannel();
-			ch.prefetch(Configuration.get('RABBITMQ_MATRIX_CONSUME_CONCURRENCY'));
```

Comment:
I have no idea what it do.. :D

Reply:
it is overriding the prefetch count, although also used for ldap, i consider that as safe